okhttp/build.gradle

```diff
@@ -29,8 +29,16 @@ dependencies {
   compileOnly deps.jsr305
   compileOnly deps.animalSniffer
 
+  testImplementation deps.okio
+  testImplementation project(':okhttp-testing-support')
+  testImplementation project(':okhttp-tls')
+  testImplementation project(':okhttp-urlconnection')
```

Comment:
This one feels strange...

Reply:
CookiesTest.java uses JavaNetCookieJar. Can fix, but not this PR.